test/testdata/asasalint.go

```diff
@@ -0,0 +1,17 @@
+package testdata
+
+import "fmt"
+
+func getArgsLength(args ...any) int {
+	return len(args)
+}
+
+func checkArgsLength(args ...any) int {
+	return getArgsLength(args)
+}
+
+func someCall() {
+	var a = []any{1, 2, 3}
+	fmt.Println(checkArgsLength(a...) == getArgsLength(a))
```

Comment:
```suggestion
	fmt.Println(checkArgsLength(a...) == getArgsLength(a)) // ERROR `pass \[\]any as any to func getArgsLength func\(args \.\.\.any\) int`
```

Reply:
LGTM

Reply:
you have to apply suggested feedback.

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request

Reply:
i haven't known this new feature, i was push new commit for update.

but from your comment, the org can not allow other's to write to. sorry again.